sensor_examples/adis16480_motion.m

```diff
@@ -0,0 +1,54 @@
+clear all; %#ok<CLALL>
+
+%% ADIS16480 Motion Example
+
+%% Setup
+% IMU
+IMU = adi.ADIS16480;
+IMU.SamplesPerRead = 8;
+IMU.uri = 'ip:analog';
+IMU.SampleRate = 128;
+IMU.EnabledChannels = 1:6;
+fs = IMU.SampleRate;
+% Filter
+ifilt = imufilter('SampleRate', fs);
+% Scopes
+N = 500;
+viewer = HelperOrientationViewer;
+% ts = dsp.TimeScope;
```

Comment:
Can you change the commented code to be uncommented and surrounded by a conditional? Less work if people want to enabled the scopes.

Reply:
I added a useScope flag to enclose the use of timescope in a conditional. useScope is true by default. The adis16460 example is updated as well.